app.js

```diff
@@ -2,9 +2,12 @@ const express = require("express");
 const app = express();
 
 const { getAllCategories } = require("./db/controllers/categories.controller");
+const { readMeFunc } = require("./db/controllers/api.controller");
 
 app.get("/api/categories", getAllCategories);
 
+app.get("/api", readMeFunc);
+
 app.get("*", (request, response, next) => {
   response.status(404).send({ message: "Path Not Found" });
```

Comment:
I would also look to now add something to handle a 500 (server error) here :) you can't really tests for this, but just ensure that it is covered :)